internal/telemetry/client.go

```diff
@@ -47,10 +49,16 @@ var (
 		},
 		Timeout: 5 * time.Second,
 	}
-	// TODO: Default telemetry URL?
 	hostname string
 )
 
+// Default values for determining where to upload telemetry
```

Comment:
Instead of doing this "group doc" can we please take the time do document them individually?

Reply:
Done